src/main/java/core/basesyntax/Lottery.java

```diff
@@ -0,0 +1,10 @@
+package core.basesyntax;
+
+import java.util.Random;
+
+public class Lottery {
+    public Ball getRandomBall(Color color) {
+        int number = new Random().nextInt(101);
+        return new Ball(color, number);
+    }
```

Comment:
this method should not have any parameters
`101` is a magic number, create constant for it
```suggestion
    public Ball getRandomBall() {
        int number = new Random().nextInt(MAX_NUMBER);
        return new Ball(colorSupplier.getRandomColor(), number);
    }
```
![image](https://github.com/mate-academy/jv-lottery/assets/73041091/af6255ee-9513-4ba7-9ebb-fc988821c4fd)